demos/bank-of-anthos-asm-manifests/demo-manifests/frontend-custom-deployment.yml

```diff
@@ -30,19 +31,18 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: front
-        # Temporarily use this container image until https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/235 is merged
```

Comment:
One problem the script has is that it doesn't keep YAML comments :/

Reply:
oof thats true. For now I can comb through this PR and make sure no other relevant/important comments are removed. I noticed that pyyaml did remove comments, I'll look into using a different module for yaml loading